saleor/core/management/commands/populatedb.py

```diff
@@ -120,3 +122,10 @@ def handle(self, *args, **options):
 
         for msg in create_permission_groups():
             self.stdout.write(msg)
+
+        if is_demo_mode():
```

Comment:
Do we really want to add a `demo things` inside the repo? I know that deploying new changes can be problematic but I believe that we can figure out some different approaches for that.

Reply:
If you really want to keep that in main repository I would suggest creating a separate file for all `demo staff` in utils directory. Just let's keep them togheter